README.md

```diff
@@ -12,9 +12,13 @@ It allows your websites to be easily installed in both Acquia Cloud IDE & local
 and deployable on Acquia Cloud.
 
 ## Installation
+###Install using Composer 
 
+```
+composer require acquia/drupal-recommended-settings
+```
 ### Include Acquia Global Command in DRS for multisite functionality
-To use global command with DRS for multisite functionality please add below in
+To setup multi-site with Drupal Recommended Settings, please add below section in
```

Comment:
Remove below section from readme file. as it will be moved to github/acquia from github/vishalkhode1, once it's been moved to github/acquia we can release a dev version.